ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/cco-manual-workload-identity/managed-identity/deprovision/cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity-managed-identity-deprovision-chain.yaml

```diff
@@ -6,6 +6,7 @@ chain:
   - ref: gather-azure-resource
   - chain: gather
   - ref: ipi-deprovision-deprovision-in-bastion
+  - ref: azure-deprovision-role-assignment
```

Comment:
Shall we rename it to `azure-deprovision-bastionhost-role-assignment`

Reply:
hmm, this step `azure-deprovision-role-assignment` is originally intended to remove any role assignment if file `"${SHARED_DIR}/azure_role_assignment_ids` exists, not only for bastion host.
If there is any identity attached on other resources assigns role on scope of subscription in the future, we only need to append role assignment id to file `azure_role_assignment_ids` and this step can be reused.

Reply:
Got it.